### DISCUSSION TITLE
Prevent image from reloading

### DISCUSSION THREAD

**Discussion by killerbytes** [TanStack/table#3852] [Q&A]
Is there a way to prevent <img> from reloading?
https://codesandbox.io/s/naughty-oskar-6qo7ej?file=/src/App.js

**Comment by tannerlinsley** [comment]
The image rerenders because technically the rows and cells have different `key`s when you paginate. rows are keyed automatically on `row.id` and cells on `cell.id`. These are all unique across the entire table model by default. You can override the keys though if you'd like to whatever you want [like this example that uses indices](https://codesandbox.io/s/tender-haslett-6n45fp?file=/src/App.js), but understand that there are potential downsides to using indices to key repeated items.
